.pre-commit-config.yaml

```diff
@@ -60,6 +60,7 @@ repos:
     hooks:
       - id: codespell
         args: ["--write-changes"]
+        exclude: ".*(CITATION.cff)$"
```

Comment:
Why is the exclusion being done in the `pre-commit` configuration and not at https://github.com/astropy/astropy/blob/302d190c8f3f03337d91b91fb8561d3be85e0999/pyproject.toml#L360-L361

Reply:
We can change it to there. I only advocated for Astropy to adopt a spell checker, someone else did the great job of setting it up. I've never used codespell outside of pre-commit, so it didn't really occur to me to configure it elsewhere.

Reply:
Done. Thanks @eerovaher.

Reply:
On a separate note, we should clean up the skip string. I wonder if it can be made into a list or some such.